.github/workflows/post-merge.yaml

```diff
@@ -38,5 +38,8 @@ jobs:
       # upload the lambda artifact to s3 and generate a cloudformation template referencing it
       - run: sam package --template-file .aws-sam/build/template.yaml --s3-bucket $ESSENTIALS_BUCKET --s3-prefix ${{ github.event.repository.name }}/${{ github.ref_name }} --output-template-file .aws-sam/build/${{ github.event.repository.name }}.yaml
 
-      # upload the generated cloudformation template to s3
+      # validate final template with cloudformation
+      - run: aws cloudformation validate-template --template-body file://.aws-sam/build/${{ github.event.repository.name }}.yaml
```

Comment:
shouldn't we `validate-template` before running `package`?

Reply:
this is running `validate-template` on the template created by `sam package`